app.js

```diff
@@ -16,6 +18,8 @@ app.set('view engine', 'jade');
 
 app.use('/', index);
 app.use('/users', users);
+app.use('/signup', signup);
```

Comment:
consider using app.get() instead of app.use() otherwise the login/signup buttons won't link to other pages... took me days to figure that one out.
```
//retrieve homepage login/signup button links  
app.get('/login', login);
app.get('/signup', signup);
```